snapcraft/extensions/gnome.py

```diff
@@ -291,6 +323,10 @@ def get_parts_snippet(self) -> Dict[str, Any]:
         """
         source = get_extensions_data_dir() / "desktop" / "command-chain"
 
+        gpu_opts = {}
+        if self.yaml_data["base"] == "core24":
+            gpu_opts["make-parameters"] = ["GPU_WRAPPER=gpu-2404-wrapper"]
```

Comment:
how married are we to the name being "wrapper"

Reply:
Not at all.

Reply:
gpu-2404 already calls this wrapper (`${SNAP}/gpu-2404/bin/gpu-2404-provider-wrapper`), so it is the right thing to call it. It only brings in personal memories from when we were full of them, so we can disregard